### DISCUSSION TITLE
README Quickstart for AgentChat is using autogen_agentchat.ui

### DISCUSSION THREAD

**Discussion by dfackler** [microsoft/autogen#4581] [General]
I believe with the latest release those should be changed to:
```
from autogen_agentchat.task import Console
from autogen_agentchat.task import TextMentionTermination
```

I ran the following and the task module is what seemed to hold Console and TextMentionTermination
```
pip install autogen-agentchat==0.4.0.dev8
pip install "autogen-ext[openai]==0.4.0.dev8"
```

**Comment by ekzhu** [comment]
Thanks for the update. Sorry we will make a release soon.

**Reply by dfackler** [reply]
All good, just wanted to share to help future users. Thanks for maintaining this!
